src/snowcli/config.py

```diff
@@ -70,10 +75,18 @@ def get_connection(self, connection_name: str) -> dict:
 
     def add_connection(self, name: str, parameters: dict):
         if not self.section_exists("connections"):
-            self._initialize_connection_section()
-        self._find_section("connections").add(name, parameters)
+            self._initialize_section("connections")
+        connections = self._find_section("connections")
+        connections.add(name, parameters)
+        if len(connections) == 1:
+            self._set_default_connection(name)
```

Comment:
I don't think we should be doing this way as this creates additional config entry. I think we should check for len = 1 when reading connections

Reply:
Right, changed